### PR TITLE
[Merged by Bors] - feat(CategoryTheory/Shift/Opposite): commutation with shifts on the opposite functor

### PR DESCRIPTION
**Pull request by smorel394**
Given a `CommShift` structure on a functor `F` between categories endowed with shifts, construct the corresponding `CommShift` structure on `F.op` for the naive shifts on the opposite categories. (And vice versa.)

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/CategoryTheory/Shift/Opposite.lean [hunk 1]
```diff
@@ -110,4 +112,67 @@ lemma oppositeShiftFunctorAdd'_hom_app :
...
+namespace CommShift
+
+open Functor
+
+/--
+Given a `CommShift` structure on `F`, this is the corresponding `CommShift` structure on
+`F.op` (for the naive shifts on the opposite categories).
+-/
+@[simps]
+noncomputable def commShiftOp [CommShift F A] :
```
Comment (joelriou) [inline Mathlib/CategoryTheory/Shift/Opposite.lean:129]: In terms of namespaces, this should be `CategoryTheory.Functor.commShiftOp` (and the one below `commShiftUnop`).

Comment (smorel394) [reply Mathlib/CategoryTheory/Shift/Opposite.lean:127]: Changed.

File: Mathlib/CategoryTheory/Shift/Opposite.lean [hunk 2]
```diff
@@ -110,4 +112,67 @@ lemma oppositeShiftFunctorAdd'_hom_app :
...
+    CommShift (C := OppositeShift C A) (D := OppositeShift D A) F.op A where
+  iso a := (NatIso.op (F.commShiftIso a)).symm
+  zero := by
+    simp only
+    rw [commShiftIso_zero]
+    ext _
```
[the remark this answers is not in the document]
Comment (smorel394) [reply Mathlib/CategoryTheory/Shift/Opposite.lean]: Ok, I removed all the underscores after `ext`.

**Comment by joelriou** [comment]
Thanks!

bors merge

**Pull request closed without merging**